OVHcloud Backint Agent with SAP HANA Cockpit (#5708)
ovh/docs: pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_configure_ovhcloud_backint_agent_several_buckets/guide.en-gb.md

OVHcloud Backint Agent for SAP HANA allows you to back up and recover your SAP HANA database on one or many OVHcloud S3 Object Storage buckets.

> Using several S3 Object Storage buckets can be useful for:

Reviewer: ```suggestion
Using several S3 Object Storage buckets can be useful to:
```

/usr/sap/<SID>/HDB<NI>/exe/hdbsql -u SYSTEM -d SYSTEMDB "BACKUP DATA FOR <SID> USING BACKINT ('MANUAL_COMPLETE_BACKUP');"
```

You also have the possibility to trigger these backups via the SAP HANA Cockpit. Select `Backint`{.action} in the `Destination Type`{.action} category.

![backup_hana_cockpit](images/backup_hana_cockpit.png){.thumbnail}

> After the execution of these backups, several files named `_databackup_` are now present in your first S3 Object Storage bucket, these files correspond to backups of your SAP HANA database via OVHcloud Backint Agent.

Reviewer: ```suggestion
After the execution of these backups, several files named `_databackup_` are now present in your first S3 Object Storage bucket, these files correspond to backups of your SAP HANA database via the OVHcloud Backint Agent.
```

> Two files named `log_backup_0_0_0_0` have `DB_<SID>` and `SYSTEMDB` prefix are also present in your first S3 Object Storage bucket. These files correspond to backups of the SAP HANA backup catalog, allowing you to list backups known by SAP HANA.

Reviewer: There's a grammar issue here.
```suggestion
Two files named `log_backup_0_0_0_0` and which have `DB_<SID>` and `SYSTEMDB` prefixes are also present in your first S3 Object Storage bucket. These files correspond to backups of the SAP HANA backup catalog, allowing you to list backups known by SAP HANA.
```

![bucket_data_backup](images/bucket_data_backup.png){.thumbnail}

> In your second S3 Object Storage bucket, are preset files named `log_backup` corresponding to logs backups.

Reviewer: ```suggestion
In your second S3 Object Storage bucket,  some files named `log_backup` are present and correspond to logs backups.
```

### Scheduling

About the backups scheduling, please refer to our guide [Install and use OVHcloud Backint Agent for SAP HANA](/pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_install_ovhcloud_backint_agent#scheduling).

### Recovery

About the recovery of your SAP HANA database, please refer to our guide [Install and use OVHcloud Backint Agent for SAP HANA](/pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_install_ovhcloud_backint_agent#recovery).

> Steps are same, even if you use different S3 Object Storage buckets for your `DATA` and `LOG` backups.

Reviewer: ```suggestion
The steps are the same, even if you use different S3 Object Storage buckets for your `DATA` and `LOG` backups.
```
